test/integration/PoolUpdatesWithHints.js

```diff
@@ -156,16 +156,16 @@ describe("PoolUpdatesWithHints", () => {
 
         // Get gas cost of transcoder()
         let tx = await bondingManager.connect(newTranscoder).transcoder(0, 0)
+        const txResNoHint = await tx.wait()
         expect(tx).to.emit(bondingManager, "TranscoderDeactivated").withArgs(transcoders[size - 1].address, dr)
```

Comment:
Should we use the `await expect(tx.hash)...` pattern here as well?

Reply:
oops, missed this one! updated.